react/src/components/Projects/ProjectListing.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Project, DesignSafeProject } from '../../types';
+import {
+  useDsProjects,
+  useProjects,
+  mergeDesignSafeProject,
+} from '../../hooks';
+import { LoadingSpinner } from '../../core-components';
+
+export const ProjectListing: React.FC = () => {
+  const { data, isLoading, isError } = useProjects();
+  let projectsData: Array<Project> = [];
+  let dsData: Array<DesignSafeProject> = [];
+  const resp = useDsProjects();
+  dsData = resp?.data?.projects ?? [];
+  projectsData = data ?? [];
+
+  const combinedProj = mergeDesignSafeProject(projectsData, dsData);
+  projectsData = combinedProj;
+
+  if (isLoading) {
+    return <LoadingSpinner />;
+  }
+
+  if (isError) {
+    return <h4>Unable to retrieve projects</h4>;
```

Comment:
"maps" is preferred over "projects" in what the user sees on frontend (even though we use that term in Geoapi). Can we match what's currently in the angular client ([here](https://github.com/TACC-Cloud/hazmapper/blob/master/angular/src/app/components/main-welcome/main-welcome.component.html#L39C24-L44)) like the "Failed to Connect to Server" and possible error information? We added more details recently (see https://github.com/TACC-Cloud/hazmapper/pull/148) regarding the error as some accounts hit some strange behavior and helps us doing support.